crates/mun_codegen/src/ir/body.rs

```diff
@@ -102,7 +102,7 @@ impl<'db, 'ink, 't> BodyIrGenerator<'db, 'ink, 't> {
             let body = self.body.clone(); // Avoid borrow issues
 
             match &body[*pat] {
-                Pat::Bind { name } => {
+                Pat::Bind { name, mode: _ } => {
```

Comment:
I think using `..` is a bit nicer here.
```suggestion
                Pat::Bind { name, .. } => {
```